warehouse/templates/email/token-compromised-leak/subject.txt

```diff
@@ -0,0 +1,17 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+
+{% extends "email/_base/subject.txt" %}
+
+{% block subject %}Your API credentials from {{ request.registry.settings["site.name"] }} were leaked{% endblock %}
```

Comment:
I think "were leaked" is a little too strong here, it makes it seem like possibly _all_ API tokens were leaked. Something like "were found on GitHub" is more clear and less scary.

Reply:
is it OK if I use "found publicly" instead ? The whole email is made so that is can be used if we acquire new sources of compromise tokens.